Runtime/Provider/MicrosoftSapiTextToSpeechProvider.cs

```diff
@@ -82,9 +82,9 @@ public void SetConfig(TextToSpeechConfiguration configuration)
         {
             this.configuration = configuration;
         }
-
+        
         /// <inheritdoc />
-        public async Task<AudioClip> ConvertTextToSpeech(string text)
+        public Task<AudioClip> ConvertTextToSpeech(string text)
```

Comment:
I don't know if removing async keyword is an API break. @Gusinuhe?

Reply:
![image](https://user-images.githubusercontent.com/3699271/89383998-b7fb6f80-d6fd-11ea-9666-704cbdd81c56.png)
Does not look like it breaks something at least

Reply:
https://stackoverflow.com/questions/13243975/suppress-warning-cs1998-this-async-method-lacks-await
This also sounds like it is fine